Bot/TynibotHost.cs

```diff
@@ -77,9 +79,9 @@ public TynibotHost()
                 var DefaultCommands = new List<Type>()
                 {
                     //typeof(Clear),
-                    typeof(Discord.Mafia.MafiaCommand),
-                    typeof(Discord.Matches.MatchesCommand),
-                    typeof(Discord.Inhouse.InhouseCommand),
+                    typeof(MafiaCommand),
+                    typeof(MatchesCommand),
+                    typeof(InhouseCommand),
```

Comment:
These were giving build errors, although I'm not entirely sure why. VS would even suggest you could use using statement, or fqcn, but the fqcn was failing at build. Moved to using statements for the few places we had fqcn's.